sql/sql_table.cc

```diff
@@ -11569,6 +11569,19 @@ bool Sql_cmd_create_table_like::execute(THD *thd)
   }
 #endif
 
+#ifdef WITH_WSREP
+  if (WSREP(thd) && select_lex->item_list.elements &&
```

Comment:
I moved select_lex->item_list.elements first. 
However, notice that WSREP(thd) is `WSREP_PROVIDER_EXISTS_ && thd->variables.wsrep_on`, this is not more expensive than the other clauses. Actually I would expect that the majority of mariadb deployments do not use wsrep at all. It would be best in that case to just evaluate WSREP_PROVIDER_EXISTS_ to false and skip the rest.